website/docs/r/access_policy.html.markdown

```diff
@@ -77,6 +77,8 @@ conditions which can be applied. The conditions are:
   `email = ["test@example.com"]`
 * `email_domain` - (Optional) A list of email domains. Example:
   `email_domain = ["example.com"]`
+* `service_token` - A list of service token ids. Example:
+  `service_token = ["${cloudflare_access_service_token.demo.id}"]`
```

Comment:
This example should be using the 0.12 syntax for all new documentation.

Reply:
(or a simple string value for the purposes of examples)

Reply:
How about we create a separate issue for migrating 0.12 syntax?

Reply:
What would the driver be for not doing it in this given that we’re introducing the change now and we aim to document 0.12 for the examples as that is the latest supported by Terraform?

Reply:
Good to know that we need to proceed for 0.12 syntax but this PR is just about to add a new parameter. 

If we are gonna to switch to 0.12 syntax, how about we do that as one shot and update on all the documents?
That could keep the document consistent and cleaner to read.

The idea is just to keep the PR simple.

Reply:
I’m not sure I follow here. The documentation should already be 0.12 compatible as a sweep was done following the 2.x release. If there is something that was missed, please drop it below as we will need to fix it.

Reply:
Ok, updated to 0.12 syntax.